lib/scrapers/human_phenotype_ontology.rb

```diff
@@ -24,6 +24,41 @@ def self.scrape_all()
       puts(i)
     end
 
+    def self.update
+      resp = Util.make_get_request(url())
+      xml = Nokogiri::XML(resp)
+      xml.xpath("//owl:Class[@rdf:about]").each do |row|
+        about = row.attributes['about'].value
+        if about.starts_with?('http://purl.obolibrary.org/obo/HP_')
+          hpo_id = about[/http:\/\/purl\.obolibrary\.org\/obo\/(HP_[0-9]*)/, 1]
+          hpo_id = hpo_id.sub('_', ':')
+          name = row.at_xpath('rdfs:label').text
+          if !row.at_xpath('owl:deprecated').nil? && row.at_xpath('owl:deprecated').text == 'true'
+            p = Phenotype.find_by(hpo_id: hpo_id)
+            if !p.nil?
+              if p.evidence_items.count == 0 && p.assertions.count == 0
+                p.delete
+              else
+                civicbot_user = User.find(385)
+                (p.evidence_items + p.assertions).each do |obj|
+                  if obj.flags.select{|f| f.state == 'open' && f.comments.select{|c| c.title == "Deprecated HPO term" && c.user_id = 385}.count > 0}.count == 0
+                    result = Flag.create_for_flaggable(civicbot_user, obj, nil)
+                    if result.succeeded?
+                      Comment.create({title: 'Deprecated HPO term', text: "This entity uses a deprecated HPO term \"#{name}\" (#{hpo_id})", user: civicbot_user, commentable: result.flag})
```

Comment:
You don't need to nest this all in a hash, `Comment.create(title: 'foo', text: 'bar'... etc)` is sufficient.